### ISSUE TITLE
Site is down

### ISSUE THREAD

**Issue by hnasheralneam** [Jerenaux/westward#19]
The link to the website does not work.

**Comment by Jerenaux** [comment]
Yes I know, I had to stop paying the hosting fees, it didn't make sense anymore for one player a month. Hopefully the code is still of value and allows people to run the game locally.

**Comment by hnasheralneam** [comment]
You could easily set it up on a Heroku hosted site. I could even put it up, if you don't mind.

**Comment by Jerenaux** [comment]
That's a good point, it used to be on Heroku actually, but I moved it because on free instances, the 'start up' time (for when a player connects after some time of inactivity) was undermining the experience. Now however it wouldn't be a problem. 
You are more than welcome to set it up yourself, thanks a lot for offering. If you need some help let me know!

**Comment by ilionic** [comment]
@Jerenaux maybe worth removing references to the site, it's currently leading to some scams.

**Comment by Jerenaux** [comment]
Done, thanks for pointing out!

**Comment by Jerenaux** [comment]
@Squirrel-314 I had a look at Heroku and as far as I can see there is no free MongoDB add-on anymore, so for now I don't see free option to host the game, but I'm open to suggestions!
